extra_data/components.py

```diff
@@ -1446,6 +1446,9 @@ class JUNGFRAU(MultimodDetectorBase):
     n_modules: int
       Number of detector modules in the experiment setup. Default is
       None, in which case it will be estimated from the available data.
+    st_modno: int
```

Comment:
I think `st` is a bit too cryptic (and it makes me think of the [results from stat()](https://docs.python.org/3/library/os.html#os.stat_result)). Let's make it a bit longer, like `start_modno` or `first_modno`.

Reply:
I like `first_modno` [4ace38c02a92a8aa7db6ac363f9865cbf4386565](https://github.com/European-XFEL/EXtra-data/pull/379/commits/4ace38c02a92a8aa7db6ac363f9865cbf4386565)